src/chocolatey.tests/infrastructure.app/services/RegistryServiceSpecs.cs

```diff
@@ -23,6 +23,7 @@ namespace chocolatey.tests.infrastructure.app.services
     using Moq;
     using Should;
     using Registry = chocolatey.infrastructure.app.domain.Registry;
+	using NUnit.Framework;

```

Comment:
Please remove this - won't be necessary when you put back `[WindowsOnly]` and remove `[Platform(Exclude)]` as instructed for the change to WindowsOnly.

Reply:
Done